mapbox/validation.py

```diff
@@ -0,0 +1,31 @@
+
+class MapboxValidationError(ValueError):
+    pass
+
+
+class InvalidPlaceTypeError(MapboxValidationError):
```

Comment:
Also I never noticed the "built-in" part, does that mean we should never raise ValueError from our modules?